pkg/oauth/apiserver/apiserver.go

```diff
@@ -27,8 +25,8 @@ import (
 )
 
 var (
-	scheme = runtime.NewScheme()
-	codecs = serializer.NewCodecFactory(scheme)
+	scheme = legacyscheme.Scheme
```

Comment:
is it okay to refer to `legacyscheme.Scheme` ? Or do we prefer to plug a new one instead ?

Reply:
custom